ignite/metrics/gan/fid.py

```diff
@@ -62,11 +62,11 @@ class FID(_BaseInceptionMetric):
     where :math:`\mu_1` and :math:`\sigma_1` refer to the mean and covariance of the train data and
     :math:`\mu_2` and :math:`\sigma_2` refer to the mean and covariance of the test data.
 
-    More details can be found in `Heusel et al. 2002`__
+    More details can be found in `Heusel et al. 2018`__
```

Comment:
arxiv references it as 2017, let's put 2017 even if the latest revision version is 2018